### PR TITLE
kie-issues#130: chrome-extension: Kogito icon is not working correctly after using the browser "back"

### LINKED ISSUE #3732: chrome-extension: Kogito icon is not working correctly after using the browser "back"
The Kogito icon becomes inaccessible after using the browser "back" in a bpmn/dmn file in the bpmn/dmn chrome-extension. Also, opening a bpmn/dmn file after the browser "back" a second Kogito icon will appear.

https://user-images.githubusercontent.com/24302289/222573699-7e2eaf56-f487-4e6b-9127-515b8139f32d.mp4

---

https://user-images.githubusercontent.com/24302289/222573716-a0864b7d-f438-4c00-8067-40495bb4e139.mp4

### PR DESCRIPTION
**Pull request by klbowers**
closes https://github.com/apache/incubator-kie-tools/issues/3732

Blocked by: 
- https://github.com/apache/incubator-kie-tools/issues/3892

### CODE REVIEW

File: packages/chrome-extension/src/app/utils.ts [hunk 1]
```diff
@@ -161,3 +162,13 @@ export function waitForElementToBeReady(selector: string) {
...
+
+// Necessary because GitHub apparently "caches" DOM structures between changes on History.
+// Without this method you can observe duplicated elements when using back/forward browser buttons.
+export function cleanupDuplicateElements(id: string, selectors: string[]) {
+  selectors.forEach((selector) => {
+    Array.from(document.querySelectorAll(`.${selector}.${id}`)).forEach((e) => {
+      removeAllChildren(e);
+    });
+  });
+}
```
Comment (ljmotta) [inline packages/chrome-extension/src/app/utils.ts:174]: Currently this function is receiving classes and building a selector in a strictly way. You could improve this function by building the selectors before passing it down to the function. This way the caller can pass down a different selector if required.

Comment (klbowers) [reply packages/chrome-extension/src/app/utils.ts:174]: Good point thanks Luiz, will update it accordingly

**Comment by klbowers** [comment]
After merging with main, the fix no longer works due to a bug https://github.com/apache/incubator-kie-tools/issues/3892. Probably best not to merge until the bug is resolved.

**Comment by jomarko** [comment]
may I ask you to include me into reviewers once ready? I would like to doble check effect of the changes for the different github instances

**Comment by tiagobento** [comment]
Closing due to inactivity.

**Pull request closed without merging**